rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -210,6 +230,76 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         cli.rate_limiting.rate_limit_generic.clone(),
     ));
 
+    // HTTP Logs Anonymization
+    let salt = Arc::new(ArcSwapOption::<Vec<u8>>::empty());
```

Comment:
Maybe call it a bit more distinctive e.g. `log_salt` and create a separate type for it that wraps `Vec<u8>`?

Reply:
@blind-oracle Is `anonymization_salt` OK for you?